src/bundle/Resources/config/routing.yml

```diff
@@ -336,6 +336,14 @@ ezplatform.content_type.list:
     requirements:
         contentTypeGroupId: \d+
 
+ezplatform.content_type.draft_list:
+    path: /contenttypegroup/{contentTypeGroupId}/contenttype/draftlist
+    methods: ['GET']
+    defaults:
+        _controller: 'EzPlatformAdminUiBundle:ContentType:draftList'
```

Comment:
Be careful on merge up, this is not gonna work in Symfony 5. [Bundle controller syntax had been deprecated in 4.1](https://symfony.com/blog/new-in-symfony-4-1-deprecated-the-bundle-notation) and was dropped in 5.0.
Though you're gonna get conflict on merge up, so you'll probably notice that.